TopMovies/Model/MovieModel.swift

```diff
@@ -9,12 +9,22 @@ import Foundation
 
 struct Movie {
     let id: ID
+    let adult: Bool
     let title: String
     let description: String
-    let rating: Float
+    let rating: Double
     let poster: URL?
     
     struct ID: Hashable {
         let value: String
     }
+    
+    init(dto: MovieDTO) {
+        id = ID(value: String(dto.id))
+        adult = dto.adult
+        title = dto.title
+        description = dto.overview
+        rating = dto.voteAverage
+        poster = URLManager.moviePosterURLFor(path: dto.posterPath)
+    }
```

Comment:
Move it in DTO model, since you may have different DTO's that will be converted in a single model, so you don't wanna mix them in a single file.

Reply:
As an extension and you can even place it in a separate file